design-library/src/components/BccGraphic/BccGraphicPoster.vue

```diff
@@ -1,23 +1,60 @@
 <script setup lang="ts">
+import { ref } from "vue";
+
 type Props = {
   bannerSrc?: string;
   logoSrc?: string;
   rounding?: "md" | "xl";
+  ratio?: keyof typeof size;
```

Comment:
This isn't passed from the parent `BccGraphic` component so there's no way for a user to customize this. Is that intended?